### PR TITLE
LON-1286

### PR DESCRIPTION
**Pull request by dfernandezramos**

### CODE REVIEW

File: VAS.Core/ViewModel/DashboardButtonVM.cs [hunk 1]
```diff
@@ -719,6 +718,10 @@ public bool Start (List<DashboardButtonVM> buttons)
 		/// <param name="buttons">Buttons.</param>
 		public void Stop (List<DashboardButtonVM> buttons)
 		{
+			if (currentNode == null) {
+				return;
+			}
+
```
Comment (SandraPG85) [inline VAS.Core/ViewModel/DashboardButtonVM.cs]: is it possible to add a quick test to ensure that no exception happens if current node is null?
